src/matchmaker/match.js

```diff
@@ -32,14 +28,14 @@ function createMatch({ player1, player2 }) {
   }
 
   const match = {
-    id: currentMatchId,
+    id: matchId,
     player1: { ...player1, matchIds: player1.matchIds.concat(matchId) },
```

Comment:
I like the idea to save the player in the match temporary.
But after the update player function where we create the new updated players there is still this "old" player saved in the matches and I think we don't need to save the whole player in match at the end of the drawfunction.

I would add a update match function after the update player function where we remove all parameters of the player  except the id

Reply:
That's a good point. Let's talk about this on wednesday!